wasm/src/main/scala/ir2wasm/WasmExpressionBuilder.scala

```diff
@@ -1735,4 +1738,35 @@ private class WasmExpressionBuilder private (
 
     IRTypes.AnyType
   }
+
+  private def genClone(t: IRTrees.Clone): IRTypes.Type = {
+    val expr = fctx.addSyntheticLocal(TypeTransformer.transformType(t.expr.tpe)(ctx))
+    genTree(t.expr, IRTypes.ClassType(IRNames.ObjectClass))
```

Comment:
It won't actually change anything but it would be more accurate to use `IRNames.CloneableClass` here.

Reply:
Didn't change anything (like a comment below) but it makes more sense, thank you !